services/brig/src/Brig/Index/Options.hs

```diff
@@ -247,6 +250,12 @@ cassandraSettingsParser =
                   <> showDefault
               )
         )
+    <*> ( (optional . strOption)
+            ( long "tls-certificate-file"
```

Comment:
```suggestion
            ( long "tls-ca-certificate-file"
```

Reply:
That's indeed a bit more descriptive. I've renamed it everywhere (it also shows up in Helm charts and docs...)